document/chapters/02-background.tex

```diff
@@ -929,14 +929,109 @@ \subsection{Signed Distance Fields}
 is repeatedly clipped using signed distance values to approach a surface in a
 process called ray marching \cite{Tomczak_2012}.
 
+\section{Monte Carlo Memory Considerations}\label{sec:mc_mem}
+
+The bulk of Monte Carlo memory consumption in native calculations is attributed
+to nuclear cross-section data and tally data structures with the analytic CSG
+geometry representation taking up a small portion of the overall program memory.
+
+\subsection{Cross-Section Data}
+
+Cross-section data is necessary to represent the physical
+processes governing particle transport through the virtual geometry. Various
+formats of this data exist with both continuous and discrete representation of
+cross section information. Continuous representations of data are compact in
+terms of memory requirements, but are more computationally expensive to evaluate
+when sampling these probability distribution functions. Discrete representations
+of this data consume more memory in simulation, but require less computation
+when extracting data. The amount of memory consumed for nuclear data is
+dependent on the number of unique materials used in the problem and material
```

Comment:
Typically depends on the number of different nuclides present as well as possible the number of materials.  There is a memory-speed trade off here as well.